src/components/AnnotationTarget/AnnotationTarget.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import noop from 'lodash/noop';
+import { KEYS } from 'box-ui-elements/es/constants';
+
+type Props = {
+    annotationId: string;
+    children: React.ReactNode;
+    className?: string;
+    onSelect?: (annotationId: string) => void;
+};
+
+const AnnotationTarget = (props: Props, ref: React.Ref<HTMLAnchorElement>): JSX.Element => {
+    const { annotationId, children, className, onSelect = noop, ...rest } = props;
+    const cancelEvent = (event: React.SyntheticEvent): void => {
+        event.preventDefault();
+        event.stopPropagation();
+        event.nativeEvent.stopImmediatePropagation(); // Prevents document event handlers from executing
+    };
+    const handleClick = (event: React.MouseEvent): void => {
+        cancelEvent(event);
+        onSelect(annotationId);
+    };
+    const handleFocus = (event: React.SyntheticEvent): void => {
+        cancelEvent(event);
+        onSelect(annotationId);
+    };
+    const handleKeyPress = (event: React.KeyboardEvent): void => {
+        if (event.key !== KEYS.enter) {
+            return;
+        }
+
+        cancelEvent(event);
+        onSelect(annotationId);
+    };
+
+    return (
+        <a
```

Comment:
should this be a button instead?

Reply:
I don't think that works as a click target inside an SVG. I can leave a comment explaining the choice.